### PR TITLE
Initial keyboard navigation

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: src/core/navigation/navigation.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+			commands = {
+				goDown: new Command({
+					shortcut: 'down',
+					canExecute: () => model().row < model().rowCount,
+					execute: () => {
+						model({row: model().row + 1});
+					}
+				}),
+				goUp: new Command({
+					shortcut: 'shift+tab|up',
```
Comment (vkorolev) [inline src/core/navigation/navigation.js]: shift + tab is opposite to tab. So  i think shift + tab should be in goLeft command

File: src/core/infrastructure/shortcut.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// import EventListener from './event.listener'; - TODO:: use instead of document
+
+export default function Shortcut(document) {
```
Comment (vkorolev) [inline src/core/infrastructure/shortcut.js]: Can you please briefly describe shortcut contract in docs? I mean, describe delimiters ( | +) and a list of encoded values

File: src/themes/default/assets/body.scss [hunk 3]
```diff
@@ -4,62 +4,70 @@
...
 
 .viewport {
-	table {
-		tbody {
-			> tr {
-				height: $row-height;
+  table {
+    tbody {
+      > tr {
+        height: $row-height;
```
Comment (vkorolev) [inline src/themes/default/assets/body.scss]: Remove spaces please :)

File: docs/navigation.md [hunk 4]
```diff
@@ -1,12 +1,48 @@
...
+
+|   Object      | Individual      | Contiguous                                                                        |
+| ------------- |:-------------:  | -----                                                                             |
+| Rows          | `shift+space`   | `shift+arrows` from current position (up/right and down/left functional equivalent) |
+| Cells         | 	`space`       | `shift+arrows` from current position                                                |
+| Columns       | `control+space` | `shift+arrows` from current position (up/right and down/left functional equivalent) |
+| Grid          | `control+A` to select the entire grid                                                               |
+
+### `Shift+F8`
+Adds additional discontinuous items to the selection set. If the user has already selected a column then shift+f8 will add additional columns to the set. Similar behavior for rows and cells.
```
Comment (klumba12) [inline docs/navigation.md]: terrific job!

File: src/ng/components/body/td.core.js [hunk 5]
```diff
@@ -23,6 +23,14 @@ class TdCore extends Directive(TD_CORE_NAME, {view: `^^${VIEW_CORE_NAME}`}) {
 		}
 
 		this.mode('init');
+		this.view.model.navigationChanged.on(e => {
```
Comment (klumba12) [inline src/ng/components/body/td.core.js]: need to think how to move it from td.core...

File: src/themes/default/assets/body.scss [hunk 6]
```diff
@@ -4,62 +4,70 @@
...
 
-				td {
-					color: $dark;
-					font-size: 13px;
-					border-top: 1px $dark-divider solid;
-					padding: 0 12px;
+        td {
+          color: $dark;
+          font-size: 13px;
+          border: {
```
Comment (klumba12) [inline src/themes/default/assets/body.scss]: don't need borders i guess?

Comment (etokareva) [reply src/themes/default/assets/body.scss]: I think, that they can be useful for future if we will change cells' styles

File: src/themes/default/assets/body.scss (hunk 3 above)
Comment (etokareva) [reply src/themes/default/assets/body.scss]: this is auto-refactoring)
Have we got tabs?

**Review by klumba12 — APPROVED** [review]

**Pull request merged**